themes/kubefest/layouts/partials/session.html

```diff
@@ -0,0 +1,9 @@
+{{ $context := . }}
+{{ $session := $context.Site.GetPage (printf "/sessions/%s" (.Scratch.Get "sessionID")) }}
+{{ $speakers := slice }}
+{{ range $id := $session.Params.speakers }}
+  {{ $speaker := $context.Site.GetPage (printf "/speakers/%s" $id) }}
+  {{ $speakers = $speakers | append $speaker.Params.name }}
```

Comment:
会社名も表示したいです！

```suggestion
  {{ $speakers = $speakers | append (printf "%s（%s）"  $speaker.Params.name $speaker.Params.company) }}
```

![image](https://user-images.githubusercontent.com/230185/82119560-8fc53a00-97ba-11ea-88b2-08a95ba8cd81.png)